pkg/cmd/subcommand/demo.go

```diff
@@ -0,0 +1,270 @@
+package subcommand
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/OpenFunction/cli/pkg/client"
+	"github.com/OpenFunction/cli/pkg/components/common"
+
+	"github.com/OpenFunction/cli/pkg/cmd/util"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	k8s "k8s.io/client-go/kubernetes"
+)
+
+// Demo is the commandline for 'init' sub command
+type Demo struct {
+	genericclioptions.IOStreams
+
+	Verbose             bool
+	RegionCN            bool
+	OpenFunctionVersion string
+	DryRun              bool
+	Delete              bool
+	Timeout             time.Duration
+}
+
+func init() {
+	w = os.Stdout
+	availableVersions = []string{"v0.3.1", "v0.4.0", "latest"}
+}
+
+// NewDemo returns an initialized Init instance
+func NewDemo(ioStreams genericclioptions.IOStreams) *Demo {
+	return &Demo{
+		IOStreams: ioStreams,
+	}
+}
+
+func NewCmdDemo(restClient util.Getter, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	var cl *k8s.Clientset
+
+	i := NewDemo(ioStreams)
+
+	cmd := &cobra.Command{
+		Use:                   "demo <args>...",
+		DisableFlagsInUseLine: true,
+		Short:                 "Create OpenFunction demo.",
+		Long: `
+This command will help you to Run the OpenFunction demo.
```

Comment:
The description could be shorter:
```
You can use ofn demo to run the OpenFunction demo.
Available options are: --delete, --region-cn, --version, --verbose, --dry-run
```

or put the options in `ofn demo -h/--help`
```
You can use ofn demo to run the OpenFunction demo.
```